2013-08-23 07:48:56 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '' for key 'username'
Request URL: /wfac/users/add
Stack Trace:
#0 /var/www/wfac/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /var/www/wfac/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('INSERT INTO `wf...', Array)
#2 /var/www/wfac/lib/Cake/Model/Datasource/DboSource.php(1004): DboSource->execute('INSERT INTO `wf...')
#3 /var/www/wfac/lib/Cake/Model/Model.php(1745): DboSource->create(Object(User), Array, Array)
#4 /var/www/wfac/lib/Cake/Controller/Scaffold.php(245): Model->save(Array)
#5 /var/www/wfac/lib/Cake/Controller/Scaffold.php(394): Scaffold->_scaffoldSave(Object(CakeRequest), 'add')
#6 /var/www/wfac/lib/Cake/Controller/Scaffold.php(153): Scaffold->_scaffold(Object(CakeRequest))
#7 /var/www/wfac/lib/Cake/Controller/Controller.php(535): Scaffold->__construct(Object(UsersController), Object(CakeRequest))
#8 /var/www/wfac/lib/Cake/Controller/Controller.php(494): Controller->_getScaffold(Object(CakeRequest))
#9 /var/www/wfac/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#10 /var/www/wfac/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /var/www/wfac/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2013-08-24 11:31:56 Error: Fatal Error (1): Call to a member function create() on a non-object in [/var/www/wfac/app/Controller/ControllerFilesController.php, line 8]
2013-08-24 11:31:56 Error: [FatalErrorException] Call to a member function create() on a non-object
Request URL: /wfac/controller_files/add
Stack Trace:
#0 /var/www/wfac/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/wfac/a...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/wfac/a...', 8, Array)
#2 /var/www/wfac/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/wfac/a...', 8, Array)
#3 /var/www/wfac/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-08-24 11:32:34 Error: Fatal Error (1): Call to a member function find() on a non-object in [/var/www/wfac/app/Controller/ControllerFilesController.php, line 22]
2013-08-24 11:32:34 Error: [FatalErrorException] Call to a member function find() on a non-object
Request URL: /wfac/controller_files/add
Stack Trace:
#0 /var/www/wfac/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/wfac/a...', 22)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/wfac/a...', 22, Array)
#2 /var/www/wfac/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/wfac/a...', 22, Array)
#3 /var/www/wfac/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-08-24 11:33:14 Error: [MissingViewException] View file "/var/www/wfac/app/View/ControllerFiles/add.ctp" is missing.
Exception Attributes: array (
  'file' => '/var/www/wfac/app/View/ControllerFiles/add.ctp',
)
Request URL: /wfac/controller_files/add
Stack Trace:
#0 /var/www/wfac/lib/Cake/View/View.php(468): View->_getViewFileName(NULL)
#1 /var/www/wfac/lib/Cake/Controller/Controller.php(952): View->render(NULL, NULL)
#2 /var/www/wfac/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#3 /var/www/wfac/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(ControllerFilesController), Object(CakeRequest), Object(CakeResponse))
#4 /var/www/wfac/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-08-24 11:34:25 Error: [MissingViewException] View file "/var/www/wfac/app/View/ControllerFiles/add.ctp" is missing.
Exception Attributes: array (
  'file' => '/var/www/wfac/app/View/ControllerFiles/add.ctp',
)
Request URL: /wfac/controller_files/add
Stack Trace:
#0 /var/www/wfac/lib/Cake/View/View.php(468): View->_getViewFileName(NULL)
#1 /var/www/wfac/lib/Cake/Controller/Controller.php(952): View->render(NULL, NULL)
#2 /var/www/wfac/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#3 /var/www/wfac/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(ControllerFilesController), Object(CakeRequest), Object(CakeResponse))
#4 /var/www/wfac/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-08-24 14:24:15 Error: Fatal Error (1): Using $this when not in object context in [/var/www/wfac/app/Model/ControllerFile.php, line 18]
2013-08-24 14:24:15 Error: [FatalErrorException] Using $this when not in object context
Request URL: /wfac/controller_files/add
Stack Trace:
#0 /var/www/wfac/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Using $this whe...', '/var/www/wfac/a...', 18)
#1 [internal function]: ErrorHandler::handleError(1, 'Using $this whe...', '/var/www/wfac/a...', 18, Array)
#2 /var/www/wfac/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Using $this whe...', '/var/www/wfac/a...', 18, Array)
#3 /var/www/wfac/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-08-24 14:39:41 Error: Fatal Error (1): Using $this when not in object context in [/var/www/wfac/app/Model/ControllerFile.php, line 18]
2013-08-24 14:39:41 Error: [FatalErrorException] Using $this when not in object context
Request URL: /wfac/controller_files/add
Stack Trace:
#0 /var/www/wfac/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Using $this whe...', '/var/www/wfac/a...', 18)
#1 [internal function]: ErrorHandler::handleError(1, 'Using $this whe...', '/var/www/wfac/a...', 18, Array)
#2 /var/www/wfac/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Using $this whe...', '/var/www/wfac/a...', 18, Array)
#3 /var/www/wfac/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-08-24 16:20:47 Error: [MissingTableException] Table my_users for model MyUser was not found in datasource default.
Exception Attributes: array (
  'table' => 'my_users',
  'class' => 'MyUser',
  'ds' => 'default',
)
Request URL: /wfac/my_users
Stack Trace:
#0 /var/www/wfac/lib/Cake/Model/Model.php(3247): Model->setSource('my_users')
#1 /var/www/wfac/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#2 /var/www/wfac/lib/Cake/Model/Model.php(1401): Model->schema()
#3 /var/www/wfac/lib/Cake/Model/Model.php(1389): Model->hasField('title', false)
#4 /var/www/wfac/lib/Cake/Model/Model.php(878): Model->hasField(Array)
#5 /var/www/wfac/lib/Cake/Controller/Scaffold.php(133): Model->__get('displayField')
#6 /var/www/wfac/lib/Cake/Controller/Controller.php(535): Scaffold->__construct(Object(MyUsersController), Object(CakeRequest))
#7 /var/www/wfac/lib/Cake/Controller/Controller.php(494): Controller->_getScaffold(Object(CakeRequest))
#8 /var/www/wfac/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#9 /var/www/wfac/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(MyUsersController), Object(CakeRequest), Object(CakeResponse))
#10 /var/www/wfac/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2013-08-24 16:47:01 Error: Fatal Error (4): syntax error, unexpected T_STRING, expecting T_FUNCTION in [/var/www/wfac/app/Controller/PostsController.php, line 4]
2013-08-24 16:47:01 Error: [FatalErrorException] syntax error, unexpected T_STRING, expecting T_FUNCTION
Request URL: /wfac/posts
Stack Trace:
#0 /var/www/wfac/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/wfac/a...', 4)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/wfac/a...', 4, Array)
#2 /var/www/wfac/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/wfac/a...', 4, Array)
#3 /var/www/wfac/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-08-24 17:04:35 Error: [MissingTableException] Table posts for model Post was not found in datasource default.
Exception Attributes: array (
  'table' => 'posts',
  'class' => 'Post',
  'ds' => 'default',
)
Request URL: /wfac/posts
Stack Trace:
#0 /var/www/wfac/lib/Cake/Model/Model.php(3247): Model->setSource('posts')
#1 /var/www/wfac/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#2 /var/www/wfac/lib/Cake/Model/Model.php(1401): Model->schema()
#3 /var/www/wfac/lib/Cake/Model/Model.php(1389): Model->hasField('title', false)
#4 /var/www/wfac/lib/Cake/Model/Model.php(878): Model->hasField(Array)
#5 /var/www/wfac/lib/Cake/Controller/Scaffold.php(133): Model->__get('displayField')
#6 /var/www/wfac/lib/Cake/Controller/Controller.php(535): Scaffold->__construct(Object(PostsController), Object(CakeRequest))
#7 /var/www/wfac/lib/Cake/Controller/Controller.php(494): Controller->_getScaffold(Object(CakeRequest))
#8 /var/www/wfac/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#9 /var/www/wfac/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest), Object(CakeResponse))
#10 /var/www/wfac/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
